keydata.go

```diff
@@ -189,24 +189,24 @@ type KeyDataReader interface {
 	ReadableName() string
 }
 
-// hashAlg corresponds to a digest algorithm.
-type hashAlg crypto.Hash
+// HashAlg corresponds to a digest algorithm.
```

Comment:
It might be worth just adding a note as to why this exists (it's just a crypto.Hash that can be serialed to JSON and DER)